browser/src/control/Control.NotebookbarImpress.js

```diff
@@ -1461,6 +1461,12 @@ L.Control.NotebookbarImpress = L.Control.NotebookbarWriter.extend({
 					}
 				],
 				'vertical': 'true'
+			},
+			{
```

Comment:
My notebookbar now looks like this:

![image](https://github.com/user-attachments/assets/2110adef-0e88-4935-a79b-920b868327b0)

Is it expected that ~all other buttons have a label next to the icon, but not this one? I would expect the new button would look similar to the existing line button, or something like that. Thanks.

Reply:
This was in my screenshot too.
I noticed, I am looking at it. I don't get why it doesn't show up.

Reply:
also it's not in Draw....

Reply:
Now fixed